docs/examples/frameworks/pytorch_setup/README.rst

```diff
@@ -0,0 +1,33 @@
+PyTorch Setup
+===================
+
+.. IDEA: Add a link to all the sections of the documentation that have to
+.. absolutely have been read before this tutorial.
+
+**Prerequisites**: (Make sure to read the following before using this example!)
+
+* :ref:`Quick Start`
+* :ref:`Running your code`
+* :ref:`Conda`
```

Comment:
Is the comment not what is done in the `Prerequisites` section?

Reply:
Yeah that's what it does. This is to explain to people who want to add new examples what it should contain.

Reply:
Ah sorry I was reading it as _"here and idea, why don't we blah"_ rather than _"the idea behind this section is"_ :)